cmd/metricscollector/v1beta1/tfevent-metricscollector/Dockerfile

```diff
@@ -1,7 +1,25 @@
-FROM tensorflow/tensorflow:1.11.0
-RUN pip install rfc3339 grpcio googleapis-common-protos
-ADD . /usr/src/app/github.com/kubeflow/katib
-WORKDIR /usr/src/app/github.com/kubeflow/katib/cmd/metricscollector/v1beta1/tfevent-metricscollector/
+FROM python:3.9
+
+ENV TARGET_DIR /opt/katib
+ENV METRICS_COLLECTOR_DIR cmd/metricscollector/v1beta1/tfevent-metricscollector
+# tensorflow community build for aarch64
+# https://github.com/tensorflow/build#tensorflow-builds
+ENV PIP_EXTRA_INDEX_URL https://snapshots.linaro.org/ldcg/python-cache/
```

Comment:
There is TF wheel for aarch64 in the server maintained by Linaro instead of [PyPI.org](https://pypi.org/project/tensorflow/#files).

Related: https://github.com/tensorflow/io/issues/1441#issuecomment-917985191

Reply:
Thank you for the clarification!